@types/gql.d.ts

```diff
@@ -51,25 +51,31 @@ type ProjectQuery = {
     name: string;
     depth: number;
     complexity: number;
-    time: number;
```

Comment:
So we are removing `time` which I think is latency. Is the logger able to track latency? It could be a good metric to include.

Reply:
We have a timestamp property, are you thinking latency as well?

Reply:
Will create an issue for this.